flink-java/src/main/java/org/apache/flink/api/java/typeutils/TupleTypeInfo.java

```diff
@@ -159,7 +161,8 @@ public String toString() {
 	}
 
 	// --------------------------------------------------------------------------------------------
-	
+
+	@SuppressWarnings("unchecked")
 	public static <X extends Tuple> TupleTypeInfo<X> getBasicTupleTypeInfo(Class<?>... basicTypes) {
```

Comment:
I would keep the original method and add a `getBasicAndBasicValueTupleTypeInfo()` method.